src/apps/benchmarks/benchmarkKRing.c

```diff
@@ -14,20 +14,29 @@
  * limitations under the License.
  */
 #include "benchmark.h"
+#include "h3Index.h"
 #include "h3api.h"
 
 // Fixtures
 H3Index hex = 0x89283080ddbffff;
 
+H3Index pentagon;
+
 BEGIN_BENCHMARKS();
 
+setH3Index(&pentagon, 9, 4, CENTER_DIGIT);
```

Comment:
This is fine, but I don't see any issue just using the indexes of "well-known pentagons" directly in tests. If you tell me a test address is a pentagon, I'll believe you :).

Reply:
I had to first calculate the index of the pentagon :)